docs/spfx/library-component-tutorial.md

```diff
@@ -120,20 +129,6 @@ localization_priority: Priority
 1. Deploy this package in the tenant app catalog and make it tenant wide deployed by checking the ```Make this solution available to all sites in the organization``` option.
  
 1. Navigate to the web part solution folder, and open the ```package.json``` file in the root of that folder.
```

Comment:
Use bold, not code fencing, for filenames.

Reply:
Formatted to bold on file names